nemo_curator/modifiers/pii_modifier.py

```diff
@@ -17,7 +17,7 @@
 import pandas as pd
 
 from nemo_curator.modifiers import DocumentModifier
-from nemo_curator.pii.algorithm import DEFAULT_LANGUAGE
+from nemo_curator.pii.constants import DEFAULT_LANGUAGE
```

Comment:
n.p. 

Why not import it DEFAULT_MAX_DOC_SIZE here, if its just a constant? 

```suggestion
from nemo_curator.pii.constants import DEFAULT_LANGUAGE, DEFAULT_MAX_DOC_SIZE
```